stan/math/prim/mat/fun/to_matrix.hpp

```diff
@@ -2,25 +2,53 @@
 #define STAN_MATH_PRIM_MAT_FUN_TO_MATRIX_HPP
 
 #include <stan/math/prim/mat/fun/Eigen.hpp>
- // stan::scalar_type
 #include <vector>
 
 namespace stan {
   namespace math {
-
-    // matrix to_matrix(matrix)
-    // matrix to_matrix(vector)
-    // matrix to_matrix(row_vector)
+    /**
+     * Returns the matrix itself.
+     *
```

Comment:
Does this only return a matrix for matrix argument?  If so, the argument shouldn't involve `R` and `C` but should just code `Eigen::Dynamic`.  If it's used to convert vectors and row vectors to matrices, then the doc should be:  `Return the matrix constructed from the data underlying the specified argument, in column-major order.  If the argument is a matrix, the matrix itself is returned.`